src/opentom_skel/start.sh

```diff
@@ -31,6 +31,10 @@ export NANOX_YRES=`fbset -s | grep geometry | if read x x yres x; then echo $yre
 # Verify TS is calibrated
 if [ ! -f $TSLIB_CALIBFILE ]; then ts_calibrate; fi
 
+# make ". s" usable to get environment in telnet 
```

Comment:
Since this would probably be useful in almost all cases, it might be even better to just add these vars to all shells (e.g. by setting `ENV` it in `~/.profile`). What do you think?